src/components/ble/MotionService.h

```diff
@@ -34,8 +33,8 @@ namespace Pinetime {
 
       uint16_t stepCountHandle;
       uint16_t motionValuesHandle;
-      std::atomic_bool stepCountNoficationEnabled {false};
-      std::atomic_bool motionValuesNoficationEnabled {false};
+      bool stepCountNoficationEnabled {false};
+      bool motionValuesNoficationEnabled {false};
```

Comment:
I agree with those, because there is no point where the atomic is used. But please correct them to "Notification" 😄